src/main/java/com/csforge/sstable/CassandraUtils.java

```diff
@@ -0,0 +1,106 @@
+package com.csforge.sstable;
+
+import com.google.common.base.Charsets;
+import com.google.common.base.Strings;
+import com.google.common.io.CharStreams;
+import com.google.common.io.Files;
+import org.apache.cassandra.config.CFMetaData;
+import org.apache.cassandra.config.DatabaseDescriptor;
+import org.apache.cassandra.config.Schema;
+import org.apache.cassandra.cql3.QueryProcessor;
+import org.apache.cassandra.cql3.statements.CFStatement;
+import org.apache.cassandra.cql3.statements.CreateTableStatement;
+import org.apache.cassandra.db.SerializationHeader;
+import org.apache.cassandra.db.marshal.AbstractType;
+import org.apache.cassandra.db.marshal.UTF8Type;
+import org.apache.cassandra.dht.IPartitioner;
+import org.apache.cassandra.io.sstable.Descriptor;
+import org.apache.cassandra.io.sstable.metadata.MetadataComponent;
+import org.apache.cassandra.io.sstable.metadata.MetadataType;
+import org.apache.cassandra.io.sstable.metadata.ValidationMetadata;
+import org.apache.cassandra.io.util.FileUtils;
+import org.apache.cassandra.schema.*;
+import org.apache.cassandra.utils.FBUtilities;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Field;
+import java.nio.ByteBuffer;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class CassandraUtils {
+    private static final Logger logger = LoggerFactory.getLogger(CassandraUtils.class);
+    private static final AtomicInteger cfCounter = new AtomicInteger();
+
+    public static CFMetaData tableFromBestSource(String sstablePath) throws IOException, NoSuchFieldException, IllegalAccessException {
+        // TODO add CQL/Thrift mechanisms as well
+        String cqlPath = System.getProperty("sstabletools.schema", "schema.cql");
```

Comment:
I don't think this works without the schema provided in the property being in the classpath right?  I think it would be better to check if the property is set and then try to load it, otherwise use classloader to find schema.cql